server/views/pages/create/checkAnswers.njk

```diff
@@ -44,7 +43,7 @@
     <div class="govuk-grid-row">
         <div class="govuk-grid-column-full">
             {% if licence.isVariation %}
-                {{ variationDetails(licence, isInProgress) }}
+                {{ variationSummary(licence, isInProgress) }}
             {% else %}
```

Comment:
There were two macros called `variationDetails` so renamed one to be `variationSummary` to differentiate them.